src/test/java/org/folio/edge/sip2/repositories/ConfigurationRepositoryTests.java

```diff
@@ -106,19 +127,32 @@ public void canGetValidAcsStatus(Vertx vertx, VertxTestContext testContext,
   public void canRetrieveTenantConfiguration(
       Vertx vertx,
       VertxTestContext testContext, @Mock Clock clock) {
-    IResourceProvider<IRequestData> resourceProvider = new DefaultResourceProvider();
+
+    List<LinkedHashMap<String, String>> configParamsList = new ArrayList();
```

Comment:
Eclipse warning, should be:
```suggestion
    List<LinkedHashMap<String, String>> configParamsList = new ArrayList<>();
```